src/librawspeed/decoders/DngDecoderSlices.cpp

```diff
@@ -418,10 +419,9 @@ void DngDecoderSlices::decodeSlice(DngDecoderThread* t) {
     while (!t->slices.empty()) {
       DngSliceElement e = t->slices.front();
       t->slices.pop();
-      LJpegPlain l(*mFile, e.byteOffset, e.byteCount, mRaw);
-      l.mDNGCompatible = mFixLjpeg;
+      DngDecompressor d(*mFile, e.byteOffset, e.byteCount, mRaw);
```

Comment:
since the last commit already moves stuff around, it kind-of makes sense here.
but no, right now i'd love for travis to come back to life.
so to avoid this naming problem entirely, i'll *try* to take just the fix